edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/TwinConfigSource.cs

```diff
@@ -251,6 +340,51 @@ internal static void FailedGettingCachedConfig(Exception ex)
                     ex,
                     Invariant($"Failed to get local config"));
             }
+
+            internal static void LogDesiredPropertiesAfterPatch(TwinCollection twinCollection)
```

Comment:
Same comment about the size - EdgeHub twin shouldn't be as large, but maybe make this LogTrace?